hack/.golint_failures

```diff
@@ -6,7 +6,6 @@ cmd/kube-controller-manager/app
 cmd/kube-proxy/app
 cmd/kube-scheduler/app
 cmd/kubeadm/app
-cmd/kubeadm/app/apis/kubeadm/v1alpha1
```

Comment:
I'm surprised the tests are passing, IIRC we had to add this for the autogenerated funcs, but that's apprarently not the case anymore :+1: